code/modules/mob/living/carbon/emote.dm

```diff
@@ -11,7 +11,35 @@
 	key_third_person = "blinks"
 	message = "blinks."
 
-/datum/emote/living/carbon/blink_r
+/datum/emote/living/carbon/blink/run_emote(mob/user, params, type_override, intentional)
+	. = ..()
+
+	var/mob/living/carbon/human = user
+
+	if (!istype(human))
+		// No neck to snap. Sad!
+		return
```

Comment:
```suggestion
/datum/emote/living/carbon/blink/run_emote(mob/living/carbon/user, params, type_override, intentional)
	. = ..()
```
redundant check because this emote should only be available to carbons anyways

Reply:
i mean simplemobs can still *blink

they just can't get statue-murdered because they're neckless

Reply:
https://github.com/tgstation/tgstation/blob/ac446d8bad3596016217112a53fe91b8140dace2/code/modules/mob/living/carbon/emote.dm#L1-L2
you sure?

Reply:
huh yeah you're right